pom.xml

```diff
@@ -37,10 +44,9 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-compiler-plugin</artifactId>
-					<version>2.5.1</version>
 					<configuration>
-						<source>1.7</source>
-						<target>1.7</target>
+						<source>8</source>
+						<target>8</target>
```

Comment:
To support GES 10.4, we can build with JDK 7 (1.7) and run on JRE 7 or JRE 8.

Reply:
On my system, it seems to be the other way around??  i just tested deploying it to a 10.4 system and it worked fine as is (compiled with target 8).

Reply:
Is 10.4 already running with JRE 8?
Was there an issue when building with 1.7 and running with 10.4?

Reply:
I've been compiling against JDK 1.8 for my Gallery Item releases with the goal of being compatible with as many versions as possible starting with 10.7 and going backwards. In my testing, 10.4 and above are compatible with compiling at JDK 1,8.  Versions 10.3 and before don't run and are not compatible.

Reply:
Is there any issue building with JDK 7/1.7 and running on JRE 8 (10.4 and above)?
If not, we should/can continue to build with JDK 7/1.7.